x-pack/elastic-agent/_meta/config/common.p2.yml.tmpl

```diff
@@ -33,6 +33,8 @@ inputs:
 #   logs: true
 #   # enables metrics monitoring
 #   metrics: true
+#   # running processes will emit metrics into logs
+#   log_metrics: true
```

Comment:
What is the difference between `metrics` and `logs_metrics`? 

We already have the `agent.logging.metrics.enabled` settings. Should we rather use that setting?

@michalpristas WDYT?

Reply:
need to check if these settings are passed to beats

Reply:
i'm not against of reusing what we have. these settings are not forwarded to beats but used for constructing our logger.